src/main/kotlin/com/openlattice/codex/CodexService.kt

```diff
@@ -75,8 +76,10 @@ class CodexService(
     val twilioQueue = HazelcastQueue.TWILIO.getQueue(hazelcast)
     val feedsQueue = HazelcastQueue.TWILIO_FEED.getQueue(hazelcast)
 
+    val BASE_PATH = "https://api.openlattice.com"
```

Comment:
🤔

Reply:
+1 it this for setting up the call back hook?

Reply:
Yeah -- this is how we've been doing it (I just moved it into a constant in this PR) but if you have a better way to do it that would be fantastic

Reply:
I guess we could add it to twilio.yaml :/

Reply:
https://github.com/openlattice/conductor-client/pull/815